README.md

```diff
@@ -144,6 +144,28 @@ http://dockerhost:5000
 When you install docker-osx-dev, it adds an entry to your `/etc/hosts` file so
 that `http://dockerhost` works as a URL for testing your Docker containers.
 
+# docker-machine support
+
+`docker-machine` support is experimental. You can use it as the way it is used for
+`boot2docker`, but run `docker-machine env` before. So as an example, run as:
+
+```
+> docker-machine create --driver virtualbox <machine-name>
+> eval "$(docker-machine env <machine-name>)"
+> docker-osx-dev install
```

Comment:
Sorry, just realized I have one more question on this: do you have to run the `install` command every time you run `docker-machine create`? Or only the `eval` part?